Verso Books.js

```diff
@@ -0,0 +1,313 @@
+{
+	"translatorID": "bc517bd2-4835-4621-b775-917253bec93a",
+	"label": "Verso Books",
+	"creator": "Bo An",
+	"target": "^https?://(www\\.)?versobooks\\.com/books/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-06-28 08:49:20"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2021 Bo An
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	const isSubject = url.includes("subjects/");
+	if (isSubject) {
```

Comment:
We don't want to return `multiple` when there are no results to scrape, even if we're theoretically on a search results page based on the URL, because `selectItems` will throw an error. Try this:

```suggestion
	if (isSubject && getSearchResults(doc, true)) {
```